tests/calculations/test_cp.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""Tests for the `PwCalculation` class."""
+from aiida.common import datastructures
+import pytest
+
+
+@pytest.mark.parametrize('autopilot', [True, False])
+def test_cp_autopilot(
+    aiida_profile, fixture_sandbox, generate_calc_job, generate_inputs_cp, file_regression, autopilot
```

Comment:
This is something changed recently
```suggestion
@pytest.mark.parametrize('autopilot', [True, False])
@pytest.mark.use_fixtures('aiida_profile')
def test_cp_autopilot(
    fixture_sandbox, generate_calc_job, generate_inputs_cp, file_regression, autopilot
```